src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -17,13 +17,25 @@
 from azure.cli.core.commands import ExtensionCommandSource
 from azure.cli.core.commands import AzCliCommandInvoker
 from azure.cli.core.commands.events import EVENT_INVOKER_ON_TAB_COMPLETION
+from azure.cli.core.azclierror import AzCLIErrorType
+from azure.cli.core.azclierror import AzCLIError
+from azure.cli.core.command_recommender import CommandRecommender
 
 from knack.log import get_logger
 from knack.parser import CLICommandParser
 from knack.util import CLIError
 
 logger = get_logger(__name__)
 
+EXTENSION_REFERENCE = ("If the command is from an extension, "
+                       "please make sure the corresponding extension is installed. "
+                       "To learn more about extensions, please visit "
+                       "'https://docs.microsoft.com/en-us/cli/azure/azure-cli-extensions-overview'")
+
+OVERVIEW_REFERENCE = ("Still stuck? Run '{command} --help' to view all commands or go to "
+                      "'https://docs.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest' "
```

Comment:
Use a short link.

Reply:
The link is the same with the original code. I suggest we apply the short link in another pr, along with other possible links.